Index: map/itemdb.c
===================================================================
--- map/itemdb.c	(revision 14215)
+++ map/itemdb.c	(working copy)
@@ -812,7 +812,7 @@
 		char path[256];
 		FILE* fp;
 
-		sprintf(path, "%s/%s", db_path, filename[fi]);
+		sprintf(path, "%s/%s", "BoY/db", filename[fi]); // Modifié pour lire les db françaises.
 		fp = fopen(path, "r");
 		if( fp == NULL )
 		{
Index: map/mob.c
===================================================================
--- map/mob.c	(revision 14215)
+++ map/mob.c	(working copy)
@@ -3628,7 +3628,7 @@
 		char path[256];
 		FILE* fp;
 		
-		sprintf(path, "%s/%s", db_path, filename[fi]);
+		sprintf(path, "%s/%s", "BoY/db", filename[fi]); // Modifié pour lire les db françaises.
 		fp = fopen(path, "r");
 		if(fp == NULL) {
 			if(fi > 0)
