Index: itemdb.c
===================================================================
--- itemdb.c	(revision 14270)
+++ itemdb.c	(working copy)
@@ -812,7 +812,7 @@
 		char path[256];
 		FILE* fp;
 
-		sprintf(path, "%s/%s", db_path, filename[fi]);
+		sprintf(path, "%s/%s", "BoY/db", filename[fi]); // Modifié pour lire les db françaises.
 		fp = fopen(path, "r");
 		if( fp == NULL )
 		{
Index: mob.c
===================================================================
--- mob.c	(revision 14270)
+++ mob.c	(working copy)
@@ -3640,7 +3640,7 @@
 		char path[256];
 		FILE* fp;
 		
-		sprintf(path, "%s/%s", db_path, filename[fi]);
+		sprintf(path, "%s/%s", "BoY/db", filename[fi]); // Modifié pour lire les db françaises.
 		fp = fopen(path, "r");
 		if(fp == NULL) {
 			if(fi > 0)
@@ -3931,7 +3931,7 @@
 	char line[1024], path[256];
 	int i, tmp=0;
 	FILE *fp;
-	sprintf(path, "%s/%s", db_path, arc); 
+	sprintf(path, "%s/%s", "BoY/db", arc); 
 	fp=fopen(path, "r");
 	if(fp == NULL)
 	{
